dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParams.java

```diff
@@ -41,11 +41,12 @@
 import org.hisp.dhis.common.AssignedUserSelectionMode;
 import org.hisp.dhis.common.IdSchemes;
 import org.hisp.dhis.common.OrganisationUnitSelectionMode;
+import org.hisp.dhis.common.UID;
 import org.hisp.dhis.event.EventStatus;
 import org.hisp.dhis.program.ProgramStatus;
 import org.hisp.dhis.program.ProgramType;
-import org.hisp.dhis.webapi.controller.event.mapper.OrderParam;
-import org.hisp.dhis.webapi.controller.event.webrequest.OrderCriteria;
+import org.hisp.dhis.tracker.Order;
+import org.hisp.dhis.webapi.controller.event.mapper.SortDirection;
```

Comment:
Note that this class is in the `dhis-api` module. This is the class we rely on so far. There are multiple classes of this sort in the code (`org.hisp.dhis.query`, `GistQuery.Order`, ...). We could discuss whether there is a way to come up with a standardized class for ordering in DHIS2 in the backend meeting.